src/main/java/com/dev/cinema/controllers/ShoppingCartController.java

```diff
@@ -0,0 +1,46 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.models.MovieSession;
+import com.dev.cinema.models.User;
+import com.dev.cinema.models.dto.user.UserResponseDto;
+import com.dev.cinema.models.mappers.ShoppingCartMapper;
+import com.dev.cinema.service.MovieSessionService;
+import com.dev.cinema.service.ShoppingCartService;
+import com.dev.cinema.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping_carts")
```

Comment:
@Vinglish this is not fixed
```suggestion
@RequestMapping("/shopping-carts")
```